plugin/start.vim

```diff
@@ -8,13 +8,20 @@ command! -range TextCaseStartReplacingCommand <line1>,<line2>lua require("textca
 
 
 function! TextCaseSubstituteLauncher(...) range
+  " Stores the first argument as a global variable
```

Comment:
Wondering if the non-inccommand version is working. perhaps it is time to drop its support.
The feature appears at version 'nvim-0.8-dev+374-ge13dcdf16'

Reply:
Then we could definitely get rid of it because I guess there are not many people using 0.7 or less nowadays 👍